es-design-system/pages/examples/form-validation.vue

```diff
@@ -37,10 +37,13 @@
                 <b-form
                     @submit.stop.prevent="onSubmit">
                     <es-form-msg
-                        class="mt-100"
+                        class="my-450"
```

Comment:
We can probably keep the `mt-100` rather than the `my-450` here. The latter gives a ton of unnecessary space around the error message:
<img width="893" alt="Screen Shot 2023-07-13 at 2 35 10 PM" src="https://github.com/EnergySage/es-ds/assets/1350363/72155fcf-cacb-4030-86a6-f442ba65de4c">